libraries/azure_data_lake_analytics_resources.rb

```diff
@@ -0,0 +1,94 @@
+require 'azure_generic_resources'
```

Comment:
I guess these files were added because of some wrong commits ..

Reply:
Yes Sateesh .I will remove these .